onelogin_rules/onelogin_active_login_activity.yml

```diff
@@ -14,6 +14,12 @@ Reports:
 Description: Multiple user accounts logged in from the same ip address.
 Reference: https://developers.onelogin.com/api-docs/1/events/event-resource 
 Runbook: Investigate whether multiple user's logging in from the same ip address is expected. Determine if this ip address should be added to the SHARED_IP_SPACE array.
+SummaryAttributes:
+  - event_type_id
+  - actor_user_name
+  - user_name
```

Comment:
I'm a bit fuzzy  on the difference between `user_name` and `actor_user_name`, are we thinking that these might be different in the case of login activity? In the underlying rule it looks like we're only considering the `user_id` so we might want to  just stick with `user_id` and `user_name`.

Reply:
Makes sense -- removing `actor_user_name` and `actor_user_id`